src/jetstream/cnsi.go

```diff
@@ -113,8 +113,8 @@ func (p *portalProxy) DoRegisterEndpoint(cnsiName string, apiEndpoint string, sk
 		}
 		return interfaces.CNSIRecord{}, interfaces.NewHTTPShadowError(
 			http.StatusBadRequest,
-			"Failed to get endpoint v2/info",
-			"Failed to get api endpoint v2/info: %v",
+			"Failed to validate endpoint",
```

Comment:
General comment, I changed this downstream as we return the same error when failing to connect to k8s. In the long term it would be good to have endpoint specific error messages

Reply:
OK